versioned_docs/version-8.7/components/camunda-integrations/sap/csap-cli.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
This should be incorporated into 8.8 as well once complete, yes? @vobu

Reply:
yes, it should all go into 8.8 as well